src/test/components/UrlManager.test.js

```diff
@@ -316,7 +316,12 @@ describe('UrlManager', function() {
     expect(previousLocation).toEqual(window.location.href);
   });
 
-  it('persists view query string for `from-addon` data source', async function() {
+  it('persists view query string for `from-addon` data source ', async function() {
+    // This setup function doesn't add any profile to the state, so that's why
+    // it logs an error that says we don't have innerWindowID in this profile.
+    // We can safely ignore that part because we don't test this part of the
+    // codebase here. We only care about the timeline track organization.
+    jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
My only small concern is that this may hide other problems in the future.

Would there be a downside adding a profile to the state? That would make our state more "correct" too.

Reply:
Thanks for the suggestion. I feel like this is going to make the test a bit more complicated without adding too much value. The important part of this test is the url. The test will throw error if the url related stuff is failed. So we can still catch the errors without the console.error.